expressEngine/lib/parser/base.js

```diff
@@ -11,6 +11,19 @@ class Parser {
   }
 
   response = (res, data, statusCode = 200) => res.status(statusCode).json(data);
+
+  validateRequired = (data = {}, required = []) => {
+    const retVal = [];
+    required.forEach((field) => {
+      if (!(data[field] || data[field] === false)) {
```

Comment:
replace `if (!(data[field] || data[field] === false)) ` with `if (!data[field])`
Both the conditions are doing the same thing

Reply:
I can't update this as it checks both condition if data[field] exists and if data[field] has a value that is equal to "false".

Reply:
I can't update this as your case only checks if key exists, this method is to check if key exists and value is not equal to undefined or null, But yes we need to add a check here if a value is 0 as a number, will update this accordingly.